lang/cs/Org.Apache.REEF.Driver.Tests/TestTaskStates.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+using Org.Apache.REEF.Driver.Task;
+using Xunit;
+
+namespace Org.Apache.REEF.Driver.Tests
+{
+    /// <summary>
+    /// The test cases in this classes test TaskState and transitions
+    /// </summary>
+    public class TestTaskStates
+    {
+        [Fact]
+        public void TestNewToCompleteSenario()
+        {
+            var taskState = new DriverTaskState();
+            Assert.True(taskState.CurrentState.Equals(TaskTransitionState.TaskNew));
```

Comment:
I dislike using Assert.True without an message a bit, because when AppVeyor test fails, it only shows "Expected True, Actual False" and line number in stack trace. I feel it's nice to have a meaningful message in Assert. Do you think it's worth adding messages to these Asserts?